### ISSUE TITLE
Enhancement - Improve "Duplicate declaration" message for Ramda fns

### ISSUE THREAD

**Issue by seanpoulter** [ramda/repl#54]
When a user declares a variable with the same name as one of Ramda's exports they get an error message that says `ramda: Duplicate declaration "<fn>"`. For users who are new to Ramda, it is not immediately obvious that there's a conflict with one of Ramda's exports.
<br/>

### Proposal

Can we change the wording to say something like:
```
ramda: Cannot declare "<fn>" that conflicts with "<fn>" already imported from Ramda
```

If that sounds good, I'm happy to try a PR. Any tips on where to look in the code?
<br/>

### Steps to Reproduce

REPL link: https://goo.gl/68HEdc

<img src="https://user-images.githubusercontent.com/2585460/37841787-e035c908-2e96-11e8-8489-0fbe6489c539.png" width="472">

**CC** @judyc

**Comment by CrossEye** [comment]
It sounds very good.  I'm afraid I have no idea where to look, though.

**Comment by buzzdecafe** [comment]
closed by #55
